src/OllamaApiClient.cs

```diff
@@ -168,7 +173,7 @@ public Task<EmbedResponse> Embed(EmbedRequest request, CancellationToken cancell
 
 		await EnsureSuccessStatusCode(response);
 
-		await foreach (var result in ProcessStreamedChatResponseAsync(response, cancellationToken))
+		await foreach (var result in ProcessStreamedChatResponseAsync(response, cancellationToken, OutgoingJsonSerializerOptions))
```

Comment:
why would you use outgoing serializer options here?

Reply:
Good catch it's a mistake. It should be the IncomingJsonSerializerOptions.